iOS_SDK/OneSignalSDK/Source/OneSignalTracker.m

```diff
@@ -81,6 +82,10 @@ + (void)endBackgroundFocusTask {
 }
 
 + (void)onFocus:(BOOL)toBackground {
+    // Privacy consent can be set by remote params, need to wait until they are set
+    if (![OneSignal downloadedParameters])
```

Comment:
We deleted this boolean on the Android SDK can we do the same for iOS?

Reply:
given that we need the delay task to have this check, I removed it, will need to check if this is need tho